pandas/indexes/base.py

```diff
@@ -432,6 +432,17 @@ def _update_inplace(self, result, **kwargs):
         # guard when called from IndexOpsMixin
         raise TypeError("Index can't be updated inplace")
 
+    def _get_grouper_for_level(self, grouper, level):
+        # Use self (Index) as grouper if None was passed
```

Comment:
can you add a ``Parameters`` section in the doc-string. and move the in-line comment to the ``Returns`` part.